scripts/skills/injury/rf_black_eye_injury.nut

```diff
@@ -0,0 +1,75 @@
+this.rf_black_eye_injury <- this.inherit("scripts/skills/injury/injury", {
+	m = {
+		VisionModifier = -1,
+		MeleeSkillMult = 0.9,
+		RangedSkillMult = 0.9,
+	},
+	function create()
+	{
+		this.injury.create();
+		this.m.ID = "injury.rf_black_eye";
+		this.m.Name = "Black Eye";
+		this.m.Description = "A powerful blow to the eye socket left the area bruised, swollen, and discolored.";
+		this.m.Type = this.m.Type | ::Const.SkillType.TemporaryInjury;
+		this.m.DropIcon = "rf_black_eye_injury";
+		this.m.Icon = "ui/injury/rf_black_eye_injury.png";
+		this.m.IconMini = "rf_black_eye_injury_mini";
+		this.m.HealingTimeMin = 1;
+		this.m.HealingTimeMax = 3;
+	}
+
+	function getTooltip()
+	{
+		local tooltip = this.skill.getTooltip();
```

Comment:
I would suggest changing this variable's name to `local ret =`. I've been doing that in my new files too. We do that with virtually all return values. What do you think?

Reply:
I started that using `tooltip` because I saw someone else (you?) do it.
I like `ret`